### PR TITLE
Upgrade BWIPP to 2022-07-29

### PR DESCRIPTION
**Pull request by terryburton**
Requires versioning for release.

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,16 @@
 History
 =======
 
+UNRELEASED
+----------
```
Comment (adamchainz) [inline HISTORY.rst:6]: I have a release script that inserts the version comment, the "UNRELEASED" header will get in the way of that 😉
```suggestion
UNRELEASED
----------

```

**Review by adamchainz — APPROVED** [review]
Thank you for doing this! Btw I normally update every 3 months - I have a calendar alert reminding me to do so.

**Comment by adamchainz** [comment]
Released in 3.17.0!

**Comment by terryburton** [comment]
Thanks.

**Pull request merged**